diff --git a/GameEngine/CMonster.cpp b/GameEngine/CMonster.cpp
index 8ef30f8..3911b80 100644
--- a/GameEngine/CMonster.cpp
+++ b/GameEngine/CMonster.cpp
@@ -11,13 +11,16 @@
 
 
 CTexture* m_pTexuture;
+CTexture* m_pWalk;
 
 CMonster::CMonster()
 {
 	this->CreateCollider();
 	this->GetCollider()->SetOffsetPos(Vector2(0.f, 0.f));
 	this->GetCollider()->SetScale(Vector2(50.f, 50.f));
-	m_pTexuture = GETINSTANCE(CResourceManager)->LoadTexture(L"Player", L"texture\\ppap.bmp");
+
+	m_pTexuture = GETINSTANCE(CResourceManager)->LoadTexture(L"Player", L"texture\\ppap.bmp");	
+	m_pWalk = GETINSTANCE(CResourceManager)->LoadTexture(L"WPlayer", L"texture\\walk.bmp");
 }
 
 CMonster::~CMonster()
@@ -78,6 +81,21 @@ void CMonster::render(HDC _dc)
 		tBlend
 	);
 
+	AlphaBlend
+	(
+		_dc,
+		pos.x - m_pWalk->Width() / 2,
+		pos.y - m_pWalk->Height() / 2,
+		m_pWalk->Width(),
+		m_pWalk->Height(),
+		m_pWalk->GetDC(),
+		0,
+		0,
+		m_pWalk->Width(),
+		m_pWalk->Height(),
+		tBlend
+	);
+
 	//충돌체등 그리기
 	CGameObject::render(_dc);
 }
diff --git a/GameEngine/CPlayer.cpp b/GameEngine/CPlayer.cpp
index 7e8e4f2..3bfba8b 100644
--- a/GameEngine/CPlayer.cpp
+++ b/GameEngine/CPlayer.cpp
@@ -30,10 +30,10 @@ CPlayer::CPlayer()
 	this->GetCollider()->SetScale(Vector2(50.f, 50.f));
 
 	this->CreateRigidbody();
-	this->GetRigidbody()->SetGravity(false);
-	this->GetRigidbody()->SetGravityAccel(310.f);
-	this->GetRigidbody()->SetVelocityLimit(200.f);
-	this->GetRigidbody()->SetGravityVelocityLimit(300.f);
+	this->GetRigidbody()->SetGravity(true);
+	this->GetRigidbody()->SetGravityAccel(1600.f);
+	this->GetRigidbody()->SetVelocityLimit(300.f);
+	this->GetRigidbody()->SetGravityVelocityLimit(330.f);
 
 
 	this->CreateAnimator();
@@ -219,6 +219,7 @@ void CPlayer::tick()
 		GetAnimator()->Play(L"IDLE", true);
 	}
 
+		//GETINSTANCE(CCamera)->SetLook(GetPos());
 	if (IS_INPUT_PRESSED(KEY::LEFT) && canWalk)
 	{
 		this->SetFilpX(false);
@@ -233,6 +234,7 @@ void CPlayer::tick()
 		Vector2 pos = this->GetPos();
 		pos.x += 300 * DELTATIME;
 		this->SetPos(pos);
+		//GETINSTANCE(CCamera)->SetLook(GetPos());
 		//GetAnimator()->Play(L"WALK_READY", false);
 	}
 	if (IS_INPUT_PRESSED(KEY::UP) && canWalk)
@@ -275,15 +277,15 @@ void CPlayer::tick()
 		GetAnimator()->Play(L"RETURN_HOME", false);
 	}
 
-	if (IS_INPUT_TAB(KEY::X))
+	if (IS_INPUT_PRESSED(KEY::X))
 	{		
 		GetRigidbody()->SetVelocity(Vector2(0, -400.f));		
 	}
 	else
 	{
-		Vector2 pos = this->GetPos();
+		/*Vector2 pos = this->GetPos();
 		pos.y += 300 * DELTATIME;
-		this->SetPos(pos);
+		this->SetPos(pos);*/
 	}
 
 	/*if (IS_INPUT_PRESSED(KEY::LEFT))
diff --git a/GameEngine/CTitleLevel.cpp b/GameEngine/CTitleLevel.cpp
index d904f13..b9996f3 100644
--- a/GameEngine/CTitleLevel.cpp
+++ b/GameEngine/CTitleLevel.cpp
@@ -26,7 +26,7 @@ void CTitleLevel::init()
 
 	CGameObject* platform = new CPlatform();
 	platform->SetPos(Vector2(500.f, 250.f));
-	platform->SetScale(Vector2(800.f, 100.f));
+	platform->SetScale(Vector2(10.f, 100.f));
 	this->AddObject(platform, LAYER::PLATFORM);
 
 
@@ -42,9 +42,9 @@ void CTitleLevel::init()
 
 
 
-	GETINSTANCE(CCamera)->SetLook(Vector2(-2368.38f, -1436.f));
+	/*GETINSTANCE(CCamera)->SetLook(Vector2(-2368.38f, -1436.f));
 	player->SetPos(Vector2(-2575.16f, -1300.53f));
-	platform->SetPos(Vector2(-2575.16f, -1225.53f));
+	platform->SetPos(Vector2(-2575.16f, -1225.53f));*/
 
 
 	/*CGameObject* monster = new CMonster();
diff --git a/GameEngine/CUIManager.cpp b/GameEngine/CUIManager.cpp
index 7c57320..6aa5b47 100644
--- a/GameEngine/CUIManager.cpp
+++ b/GameEngine/CUIManager.cpp
@@ -93,7 +93,7 @@ CUI* CUIManager::GetPriorityUI(CUI* _pParentUI)
 			// 기존 UI 의 LBTNDown 상태를 갱신한다.
 			if (bLbtnReleased && nullptr != pPriorityUI && pPriorityUI->IsLbtnDown())
 			{
-				pUI->m_bLbtnDown = false;
+				pPriorityUI->m_bLbtnDown = false;
 			}
 
 			// 우선순위 UI 로 지정
diff --git a/GameEngine/define.h b/GameEngine/define.h
index 8dc0deb..a1c0c03 100644
--- a/GameEngine/define.h
+++ b/GameEngine/define.h
@@ -25,7 +25,7 @@
 //#define windowSizeY 460
 
 
-#define _EDITOR_MODE
+//#define _EDITOR_MODE
 
 
 #define LEVEL_EDITOR 1
@@ -34,11 +34,11 @@
 #ifdef  _EDITOR_MODE
 #define LEVEL_MODE LEVEL_EDITOR
 #define windowSizeX 1280
-#define windowSizeY 720
+#define windowSizeY 1000
 #else
 #define LEVEL_MODE LEVEL_GAME
-#define windowSizeX 720
-#define windowSizeY 460
+#define windowSizeX 960
+#define windowSizeY 720
 #endif
 
 class CEntity;
